meta-mender-qemu/scripts/docker/setup-mender-configuration.py

```diff
@@ -73,6 +74,26 @@ def main():
             rootfs=rootfs)
         os.unlink("mender.conf")
 
+    if args.server_ip:
+        if not args.server_url:
+            print("Error: --server-ip requires --server-url")
+            sys.exit(1)
+        get(local_path="hosts",
+            remote_path="/etc/hosts",
+            rootfs=rootfs)
+        with open("hosts") as fd:
+            hosts = fd.readlines()
+        hosts.append("\n")
+        hosts.append("# Added by setup-mender-configuration.py\n")
+        server_host = args.server_url.removeprefix("http://").removeprefix("https://")
+        hosts.append(f"{args.server_ip} {server_host}\n")
+        with open("hosts", "w") as fd:
+            fd.writelines(hosts)
```

Comment:
You could simplify this by using `open("hosts", "a") as fd` instead.

Reply:
Or course :sweat_smile: Thanks. I was biased by the `json` approach above.